generators/CoreCLRStatefulService/index.js

```diff
@@ -81,13 +81,33 @@ var ClassGenerator = generators.Base.extend({
     var serviceProject = path.join(appPackage , 'src' , serviceSrcPath , serviceProjName + '.csproj');
     var codePath = path.join(appPackage , appPackagePath, servicePackage, 'Code');
 
+    var is_Windows = (process.platform=='win32');
+    var is_Linux = (process.platform=='linux');
+    var is_mac = (process.platform=='darwin');
```

Comment:
Same comments as above for other types as well..

Reply:
Done.